src/main/kotlin/br/com/creditas/projethom/service/EmployeeService.kt

```diff
@@ -50,19 +63,31 @@ class EmployeeService(
         id: UUID,
         updateEmployeeRequest: EmployeeRequest
     ): EmployeeResponse {
-        val employee = employeeRepository.getReferenceById(id)
-        val newTeam = teamRepository.getReferenceById(updateEmployeeRequest.teamId)
-        employee.personId = updateEmployeeRequest.personId
-        employee.team = newTeam
-        employee.role = updateEmployeeRequest.role
-        employeeRepository.save(employee)
-        return EmployeeResponse.fromEntity(employee)
+        try {
+            val employee = employeeRepository.getReferenceById(id)
```

Comment:
Se o employee não for encontrado, provavelmente vai quebrar na próxima chamada, certo?